### PR TITLE
use labels to add metadata for marathon

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn review please
I've added the `labels` with the urls info that you'll need + do a PUT when the project is edited.

File: unicoremc/models.py [hunk 1]
```diff
@@ -436,9 +450,16 @@ def initiate_create_marathon_app(self, cmd):
             "cmd": cmd,
             "cpus": 0.1,
             "mem": 100.0,
-            "instances": 1
+            "instances": 1,
+            "labels": {
+                "domain": domain,
+                "app_type": self.app_type,
```
Comment (smn) [inline unicoremc/models.py]: Why are we adding the `app_type` twice? Once here and once in `domain.app_type` ?

File: unicoremc/models.py [hunk 2]
```diff
@@ -448,6 +469,21 @@ def initiate_create_marathon_app(self, cmd):
                 'Create Marathon app failed with response: %s - %s' %
                 (resp.status_code, resp.json().get('message')))
 
+    def update_marathon_app(self):
+        post_data = self.get_marathon_app_data()
+        app_id = post_data.pop('id')
+        resp = requests.put(
+            '%(host)s/v2/apps/%(id)s?force=true' % {
```
Comment (smn) [inline unicoremc/models.py]: I'm a bit uncomfortable with making this a `force=true` can we make that optional?

File: unicoremc/models.py (hunk 1 above)
Comment (miltontony) [inline unicoremc/models.py]: Meta data, didn't think it mattered much. Thought it might be easier for your logic further down the pipeline.

Comment (miltontony) [inline unicoremc/models.py]: I can remove it too

Comment (smn) [inline unicoremc/models.py]: Yeah, rather stick with one.

**Comment by smn** [comment]
:+1:

**Pull request merged**